modules/installation-special-config-raid-intel-vroc.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installating-customizing.adoc
+
+[id="installation-special-config-raid-intel-vroc_{context}"]
+== Configuring an Intel(R) Virtual RAID on CPU (VROC) data volume
+
+Intel(R) VROC is a type of hybrid RAID, where some of the maintenance is offloaded to the hardware, but appears as software RAID to the operating system.
+
+The following procedure configures an Intel(R) VROC-enabled RAID1.
+
+.Procedure
+
+. Create the Intel(R) Matrix Storage Manager (IMSM) RAID container by running the following command:
++
+[source,terminal]
+----
+$ mdadm -CR /dev/md/imsm0 -e imsm -n2 /dev/nvme0n1 /dev/nvme1n1
+----
+
+. Create the RAID1 storage inside the container:
+
+.. Create a dummy RAID0 volume in front of the real RAID1 volume:
```

Comment:
@jlebon do you think it's worth adding a note about the dummy array and why it has to be done? is it a workaround for a bug?